<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XGBoostClass</class>
 <widget class="QMainWindow" name="XGBoostClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XGBoost</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="optimize">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XGBoost调参</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>427</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XGBoost训练</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>调参设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>(英文)训练集路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="train_path">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>35</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>./train.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="valid_path">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>75</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>./valid.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>(英文)验证集路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="train_rows">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>训练行数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>验证行数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="valid_rows">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>特征数：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="features_cols">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>21</string>
     </property>
    </widget>
    <widget class="QPushButton" name="select_train">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>35</y>
       <width>21</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>…</string>
     </property>
    </widget>
    <widget class="QPushButton" name="select_valid">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>75</y>
       <width>21</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>…</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="model_path">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>输出模型：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>XGBoost训练工具</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>训练参数</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>max_depth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>min_child</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_child">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_depth">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>gamma</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gamma">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>subsample</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subsample">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="colsample">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>colsample</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XGBoost.qrc"/>
 </resources>
 <connections/>
</ui>
